src/forms/FormFieldAdvancedSelection/AdvancedSelectionDrawer.tsx

```diff
@@ -323,7 +337,7 @@ const AdvancedSelectionDrawer = (props: AdvanceSelectionDrawerPropTypes): ReactE
 				{
 					name: "checkboxList",
 					type: "checkbox",
-					disabled: fieldDef?.disabled,
+					disabled: checkboxListDisabled,

```

Comment:
We still want to disable the field when fieldDef.disabled === true or when checkboxListDisabled === true.

```suggestion
					disabled: fieldDef?.disabled || checkboxListDisabled,

```